include/groonga/util.h

```diff
@@ -214,6 +214,82 @@ grn_inspect_name(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj);
  */
 GRN_API grn_obj *
 grn_inspect_encoding(grn_ctx *ctx, grn_obj *buffer, grn_encoding encoding);
+/**
+ * \brief Inspect the given type and produce its name.
+ *
+ * \since 4.0.8
+ *
+ * For example usage:
+ * ```c
+ * grn_obj type_name;
+ * GRN_TEXT_INIT(&type_name, 0);
+ * grn_inspect_type(ctx, &type_name, obj->header.type);
+ * printf("%.*s\n",
+ *        (int)GRN_TEXT_LEN(&type_name),
+ *        GRN_TEXT_VALUE(&type_name));
+ * GRN_OBJ_FIN(ctx, &type_name);
+ * ```
+ *
+ * For example output:
+ * Depending on \p type, it will output one of the following lines:
+ *
+ * Table types:
+ * ```
+ * GRN_DB
```

Comment:
`GRN_DB` is not table.
So, I believe that `GRN_DB` is not table types.

Reply:
Should we move to `Other object types` here?

Reply:
I think it's good.